app/controllers/v1/spins_controller.rb

```diff
@@ -11,18 +11,54 @@ class SpinsController < ApplicationController
     # Provides an index of all spins in the system
     # TODO If you provide a search team, it will return those spins mathing the search
     # TODO: Add paging
+    # users/<user_id>/spins Get all spins of a user
+    # spins?query=<value>  Look spins include value in the name
     def index
-      @spins = Spin.all
-      render json: { data: @spins }, status: :ok
+      if params[:user_id]
+        @user = User.find_by_github_login(params[:user_id])
+        unless @user
+          return_response status: :no_content
+          return
+        end
+        @spins = Spin.where(user_id: @user.id ) if @user
+      else
+        @spins = Spin.all
```

Comment:
It is OK for now. Opening a travis use case to make sure that in the future we only provide 10 results. We don't want to return 100's

Reply:
pagination feature....